### PR TITLE
Preflight prompt fix

### PR DESCRIPTION
**Pull request by SeanTConrad**
## Why are these changes needed?

Fixed a bug where the preflight prompt was not asking a question before showing `(Y/n)` and made the prompting easier to read.

## Checks

- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

File: src/verinfast/agent.py [hunk 1]
```diff
@@ -534,19 +535,19 @@ def preflight(self):
                         if provider.provider == "aws" and self.checkDependency("aws", "AWS Command-line tool"):
                             account_id = str(provider.account).replace('-', '')
                             if find_profile(account_id, self.log) is None:
-                                self.log(tag=f"No matching AWS CLI profiles found for {provider.account}", msg="Account can't be scanned.", display=True)
+                                self.log(tag=f"No matching AWS CLI profiles found for {provider.account}", msg="Account can't be scanned.", display=True, timestamp=False)
                             else:
-                                self.log(tag="Access confirmed", msg=account_id, display=True)
+                                self.log(tag="AWs account access confirmed", msg=account_id, display=True, timestamp=False)
```
Comment (aylusltd) [inline src/verinfast/agent.py:540]: Capitalize AWS

File: src/verinfast/user.py [hunk 2]
```diff
@@ -25,7 +25,8 @@ def repeat_boolean_prompt(
         print_opts = "(Y/n)"
         d = "y"
 
-    logger(prompt)
+    prompt = "\n\n" + prompt
+    print(prompt)
```
Comment (aylusltd) [inline src/verinfast/user.py:29]: I think we wanted logger with display=True, timestamp=False so it would show in the logs.

**Review by aylusltd — CHANGES_REQUESTED** [review]

File: src/verinfast/user.py (hunk 2 above)
Comment (SeanTConrad) [reply src/verinfast/user.py]: I tried that, but logger forced a ":" to be in the log.  I updated logger. We need (in a future release) a "logger" and "pretty printer."

File: src/verinfast/agent.py (hunk 1 above)
Comment (SeanTConrad) [reply src/verinfast/agent.py]: fixed

File: src/verinfast/utils/utils.py [hunk 3]
```diff
@@ -131,14 +131,17 @@ def __init__(self, path: str, debug: bool = False):
         self.debug = debug
 
     def log(self, msg, tag=None, display=False, timestamp=True):
-        if tag is None:
+        if tag is None and timestamp:
```
Comment (aylusltd) [inline src/verinfast/utils/utils.py:134]: This prevents logging the source of the issue without a timestamp. Is that intentional?

Comment (SeanTConrad) [reply src/verinfast/utils/utils.py:134]: It is. This was the cause of the mystery long source in my "pretty" log. My theory was a log is either "pretty" (so no source or timestamp) or "full." I was also trying to get this PR done in 10 minutes yesterday to hopefully release yesterday. Now that we are moving it back we could do a tiny bit more.

**Review by aylusltd — APPROVED** [review]

**Pull request merged**